src/data/quality-assurance-quiz.ts

```diff
@@ -182,6 +182,58 @@ const qualityAssuranceQuiz = [
+      "Functional testing and non-functional testing are the same thing.",
+    Distractor2:
+      "Functional testing only involves manual testing, while non-functional testing uses automation.",
+    Distractor3: "Non-functional testing is limited to web applications.",
+    Explanation:
+      "Functional testing ensures that the software meets specified requirements, while non-functional testing assesses how well the software performs under various conditions.",
+    Link: ""
+  },
+  {
+    Question:
+      "Explain the concept of 'Test Driven Development (TDD)' in software development. What are its key principles and benefits?",
+    Answer:
+      "'Test Driven Development (TDD)' is a development approach where developers write test cases before writing the actual code. The key principles include writing failing tests first, writing the minimum code required to pass the test, and continuously refactoring code. Benefits include improved code quality, faster development, and reduced debugging time.",
+    Distractor1: "TDD involves writing tests after the code is developed.",
+    Distractor2: "TDD is only applicable to front-end development.",
+    Distractor3: "TDD is primarily used for load testing.",
+    Explanation:
+      "TDD helps ensure that code is thoroughly tested and that new changes do not break existing functionality.",
+    Link: ""
+  },
+  {
+    Question:
+      "What is 'Exploratory Testing,' and how does it differ from scripted testing? When is it most useful?",
+    Answer:
+      "'Exploratory Testing' is a testing approach where testers explore the software application without predefined test cases. They use their domain knowledge and experience to find defects and assess the software's usability. It differs from scripted testing, where tests are preplanned and documented. Exploratory testing is most useful for uncovering unexpected issues and assessing user experience.",
+    Distractor1: "Exploratory testing is primarily automated.",
+    Distractor2:
+      "Scripted testing is less structured than exploratory testing.",
+    Distractor3: "Exploratory testing is only used for security testing.",
+    Explanation:
+      "Exploratory testing is valuable for finding defects that scripted testing might miss and for evaluating the software from a user's perspective.",
+    Link: ""
+  },
+  {
+    Question:
+      "What is 'Continuous Integration (CI)' and 'Continuous Delivery (CD)' in the context of software testing and deployment? How do they improve software quality and release processes?",
+    Answer:
+      "'Continuous Integration (CI)' is the practice of frequently merging code changes into a shared repository and automatically running tests to detect integration issues. 'Continuous Delivery (CD)' extends CI by automating the deployment process, allowing for rapid and reliable software releases. They improve software quality by catching issues early and enabling faster, more reliable releases.",
```

Comment:
answer is to long. Long answers work better for the explanation field. 

```suggestion
      "'CI/CD is an automated process that involves frequent code integration, automated testing, and continuous deployment of software changes to production.",
```